app/models/app.rb

```diff
@@ -1,16 +1,17 @@
 class App < ActiveRecord::Base
   belongs_to :org
+
   has_many :comments, dependent: :destroy, as: :commentable
-  has_many :engagements
+  has_many :engagements, dependent: :destroy
```

Comment:
might want to make a note here for future teams in case they want to implement support for keeping engagement data for users even if the app they worked on has been taken out of the database. Good addition with the current architecture, though!

Reply:
Yeah sure I will at the README.md. Thanks for the comments!